public/app/features/variables/interval/actions.ts

```diff
@@ -29,22 +28,22 @@ export const updateIntervalVariableOptions = (
 };
 
 export interface UpdateAutoValueDependencies {
-  kbn: typeof kbn;
+  rangeUtil: any; // typeof rangeUtil?
```

Comment:
@hugohaggmark any ideas here?  `rangeUtil` is an import namespace, not a real object like kbn